src/libultra/gu/sins.c

```diff
@@ -1,12 +1,17 @@
 #include "ultra64.h"
 
+#include "config.h"
+#ifndef DISABLE_SIN_COS_LOOKUP_TABLE
 #include "sintable.inc.c"
-
+#endif
 /**
  * @param angle binang
  * @return sin(angle)*0x7FFF
  */
 s16 sins(u16 angle) {
+#ifdef DISABLE_SIN_COS_LOOKUP_TABLE
+return Math_SinS(angle) * 0x7FFF;
```

Comment:
this function isn't declared (just add ``#include "functions.h"``) and it returns a f32, not s16, I feel like we should add a cast here (same comment for coss for the cast)